layer/json/VkLayer_gfxreconstruct.json.in

```diff
@@ -21,39 +21,23 @@
             "presets": [
                 {
                     "label": "Initialization",
-                    "description": "Capture the applications first two frames.",
+                    "description": "Capture the first two frames",
                     "platforms": [ "WINDOWS", "LINUX" ],
                     "settings": [
-                        {
-                            "key": "capture_trigger",
-                            "value": ""
-                        },
```

Comment:
@christophe-lunarg I removed this assuming the defaults below will be used

Reply:
Erm that should remain in the manifest file. A preset identify what are the values that should be set when selected, vkconfig doesn't reset all the value to default when setting a preset.

So if the value is not in the preset, it's not set.

Reply:
Ahh that makes sense, I'll just remove the unrelated `memory_tracking_mode` then